src/test/feature/lib/sql-util.cpp

```diff
@@ -79,7 +79,7 @@ void SQLUtility::execSQLFile(const std::string &sqlFile,
 std::unique_ptr<PSQL> SQLUtility::getConnection() {
   std::unique_ptr<PSQL> psql(
       new PSQL(HAWQ_DB, HAWQ_HOST, HAWQ_PORT, HAWQ_USER, HAWQ_PASSWORD));
-  return std::move(psql);
+  return psql;
```

Comment:
Here I returned pointer ownership, otherwise this memory will be deleted by unique pointer